### ISSUE TITLE
UWP not working with Beta 7

### ISSUE THREAD

**Issue by rowanmiller** [aspnet/EntityFramework#3019]
Install is blocked by a bug in NuGet https://github.com/NuGet/Home/issues/1304.

NuGet has fixed that bug, but even with an internal build with the issue fixed we have an issue with dependencies.

> Version conflict detected for System.Collections.  App20 (≥ 1.0.0) -> EntityFramework.Core (≥ 7.0.0--) -> System.Collections (≥ 4.0.11-beta-23225)  App20 (≥ 1.0.0) -> Microsoft.NETCore.UniversalWindowsPlatform (≥ 5.0.0) -> Microsoft.NETCore.Runtime (≥ 1.0.0) -> Microsoft.NETCore.Runtime.CoreCLR-x86 (≥ 1.0.0) -> System.Collections (= 4.0.10).

We need to provide guidance to unblock users Beta 7. We also need to sort out our dependencies (not just us, but Microsoft.Framework.* that we depend on) and see if we really need to have them that high, or if we can have a lower min version.

**Comment by divega** [comment]
@bricelam is this one of the technical reasons you created PRs like https://github.com/aspnet/Caching/pull/101 a few days ago?

**Comment by bricelam** [comment]
One of them, yes. I've also seen many asks for us to support Windows/Windows Phone 8, so I was investigating the feasibility of that.

**Comment by bricelam** [comment]
See also #2892

**Comment by natemcmaster** [comment]
This may be resolved by using a newer version of Microsoft.NETCore.UniversalWindowsPlatform (as long as this newer NETCore version doesn't cause further conflicts.)

```json
"Microsoft.NETCore.UniversalWindowsPlatform": "5.0.1-beta-23225"
```

**Comment by divega** [comment]
Is this still going to be blocked (without the .NET native tool chain) in beta8? Otherwise can we close the issue?

**Comment by natemcmaster** [comment]
EF installs on UWP now, and works in "Debug" mode only. This is a separate issue from netnative support. We can close this.

**Comment by bricelam** [comment]
Resolved by #3279
